static/_redirects

```diff
@@ -84,20 +84,21 @@
 /products/clickhouse/list-overview               https://aiven.io/docs/products/clickhouse
 /products/clickhouse/concepts/features-overview  https://aiven.io/docs/products/clickhouse
 
-/products/flink/list-overview                     https://aiven.io//docs/products/flink
-/products/flink/concepts/flink-features           https://aiven.io//docs/products/flink
-/products/flink/concepts/managed-service-features https://aiven.io//docs/products/flinks
+/products/flink/list-overview                     https://aiven.io/docs/products/flink
+/products/flink/concepts/flink-features           https://aiven.io/docs/products/flink
+/products/flink/concepts/managed-service-features https://aiven.io/docs/products/flinks
 
-/products/dragonfly/concepts/overview            https://aiven.io//docs/products/dragonfly
-/products/dragonfly/concept                      https://aiven.io//docs/products/dragonfly
+/products/dragonfly/concepts/overview            https://aiven.io/docs/products/dragonfly
+/products/dragonfly/concept                      https://aiven.io/docs/products/dragonfly
 
-/products/caching/concepts/overview              https://aiven.io//docs/products/caching
+/products/caching/concepts/overview              https://aiven.io/docs/products/caching
 
-/products/grafana/concepts/grafana-features      https://aiven.io//docs/products/grafana
-/products/grafana/list-overview                  https://aiven.io//docs/products/grafana
+/products/grafana/concepts/grafana-features      https://aiven.io/docs/products/grafana
+/products/grafana/list-overview                  https://aiven.io/docs/products/grafana
 
-/products/opensearch/list-overview               https://aiven.io//docs/products/opensearch
-/products/opensearch/concepts/service-overview   https://aiven.io//docs/products/opensearch
-/products/opensearch/troubleshooting             https://aiven.io//docs/products/opensearch
+/products/opensearch/list-overview               https://aiven.io/docs/products/opensearch
+/products/opensearch/concepts/service-overview   https://aiven.io/docs/products/opensearch
+/products/opensearch/troubleshooting             https://aiven.io/docs/products/opensearch
 
-/products/metrics/concepts/metrics-overview      https://aiven.io//docs/products/metrics
+/products/metrics/concepts/metrics-overview      https://aiven.io/docs/products/metrics
+/platform/howto/metrics-integrations             https://aiven.io/docs/platform/howto/list-monitoring
```

Comment:
General comment to the idea of moving the "Logs and monitoring" section into the "Integrations" category:
Indeed all the subsections of the "Logs and monitoring" section describes integrations, but the [landing/overview page](https://3a63566f.aiven-docs.pages.dev/docs/platform/howto/list-monitoring) of the "Logs and monitoring" section itself covers general information on metrics, logs, alerts, and notifications, which is not necessarly associated with integrations and, hence, can be difficult to discover for the users under the "Integrations" category. The only subsection related to integrations on the the [landing/overview page](https://3a63566f.aiven-docs.pages.dev/docs/platform/howto/list-monitoring) of the "Logs and monitoring" section is [Export service metrics and logs](https://3a63566f.aiven-docs.pages.dev/docs/platform/howto/list-monitoring#export-service-metrics-and-logs). Perhaps we could split the landing/overview page content and keep the "Export" part under the integrations and the remaining part of the landing/overview page move outside the integrations to "Services" or "Platoform"?